imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAConfig.java

```diff
@@ -45,6 +47,8 @@
     public static final String AD_MAX_REDIRECTS         = "adMaxRedirects";
     public static final String AD_ENABLE_DEBUG_MODE     = "enableDebugMode";
     public static final String AD_OMID_ENABLED          = "isOMIDExperimentalEnabled";
+    public static final String AD_PLAYER_TYPE           = "playerType";
```

Comment:
These constants should be renamed to `*_KEY`: `AD_MAX_REDIRECTS_KEY`, `AD_PLAYER_TYPE_KEY` etc, making it obvious that the value is the key and not the actual value.